mapie/quantile_regression.py

```diff
@@ -462,65 +529,80 @@ def fit(
         MapieQuantileRegressor
              The model itself.
         """
-        # Checks
-        self._check_parameters()
-        checked_estimator = self._check_estimator(self.estimator)
-        alpha = self._check_alpha(self.alpha)
-        self.cv = self._check_cv(cast(str, self.cv))
-        X, y = indexable(X, y)
-        random_state = check_random_state(random_state)
-        results = self._check_calib_set(
-            X,
-            y,
-            sample_weight,
-            X_calib,
-            y_calib,
-            calib_size,
-            random_state,
-            shuffle,
-            stratify,
-        )
-        X_train, y_train, X_calib, y_calib, sample_weight_train = results
-        X_train, y_train = indexable(X_train, y_train)
-        X_calib, y_calib = indexable(X_calib, y_calib)
-        y_train, y_calib = _check_y(y_train), _check_y(y_calib)
-        self.n_calib_samples = _num_samples(y_calib)
-        check_alpha_and_n_samples(self.alpha, self.n_calib_samples)
-        sample_weight_train, X_train, y_train = check_null_weight(
-            sample_weight_train,
-            X_train,
-            y_train
-        )
-        y_train = cast(NDArray, y_train)
-
         # Initialization
         self.estimators_: List[RegressorMixin] = []
+        if self.cv == "prefit":
+            estimator = cast(List, self.estimator)
+            self._prefit_params(estimator, X, y, X_calib, y_calib)
+            X_calib, y_calib = indexable(X_calib, y_calib)
 
-        # Work
-        y_calib_preds = np.full(
-            shape=(3, self.n_calib_samples),
-            fill_value=np.nan
-        )
+            self.n_calib_samples = _num_samples(y_calib)
```

Comment:
`y` instead of `y_calib`.

Reply:
Same reasoning as previous comment, so keeping it as `y_calib`.